### DISCUSSION TITLE
routing.useHashRouting: false bug in Angular with Luigi

### DISCUSSION THREAD

**Discussion by andrew-tud** [SAP/luigi#2988] [Q&A] [answered]
Hi, 

I am trying to use path routing in an Angular Shell Project with Luigi Framework by configuring:
`routing: {`
      `useHashRouting: false`
    `}`

But there seems to be a bug in an Angular Template Project for Luigi Framework as when the user refreshes the default page, the website goes to the Angular initial page as such:
![image](https://user-images.githubusercontent.com/19533263/202224411-3d5df4c5-204f-4e99-ab66-02a8b05eca29.png)

and when the user refreshes again the page, it toggles to Luigi Framework Shell's default page as such:
![image](https://user-images.githubusercontent.com/19533263/202224645-37d11a04-8ae8-48ac-8f71-247d5bde9cd8.png)

Is this a bug or am I missing some configuration with `useHashRouting: false`. 

Thank you.

**Comment by ndricimrr** [comment, marked as answer]
Hi @andrew-tud ,
Indeed this is not supposed to happen when disabling hashrouting. 
I could not reproduce the specified behaviour in your question.

It could be that our simple angular setup script might be outdated. Will look into it. Meanwhile  you can try out disabling hash routing  with [this example here](https://github.com/SAP/luigi/tree/master/core/examples/luigi-example-angular), which is more up to date .
Let us know if this works for you.
